### ISSUE TITLE
Select has axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#1932]
- id attribute value must be unique (issue with headings)

**Comment by redallen** [comment]
Closing in favor of #2158.
